libs/libradiotap/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2016 Bruno Randolf (br1@einfach.org)
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libradiotap
+PKG_VERSION:=2016-11-07
```

Comment:
This date is not consistent with the revision.

Reply:
this is yyyy-mm-dd, or?